apps/emqx_dashboard/src/emqx_dashboard.erl

```diff
@@ -82,7 +82,7 @@ stop_listeners() ->
     [begin
         case minirest:stop(Name) of
             ok ->
-                ?ULOG("Stop listener ~ts on ~p successfully.~n", [Name, Port]);
+                ?ULOG("Listener ~ts on ~ts stopped.~n", [Name, emqx_listeners:format_addr(Port)]);
```

Comment:
* There are gateway listeners logged in the old style.
* I’d recommend to keep the old format as these are the lines familiar to all EMQ X users from very early days.

Reply:
But we are making changes in 5.x, maybe it would be better to unify these listener prompts?

Reply:
Prefer to keep if there is no strong argument about the wording/phrasing.

Someone might be grep-ing successfully in scripts.

Reply:
ok